app.js

```diff
@@ -1,31 +1,30 @@
-// const express = require('express');
-// const routes = require("./routes")
-// const fs = require("fs");
+const express = require('express');
+const routes = require("./routes")
+const fs = require("fs");
```

Comment:
Is `fs` used in this file? Please delete the unused imports/require-statements

Reply:
done